sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServerErrors.scala

```diff
@@ -36,11 +36,10 @@ object HiveThriftServerErrors {
       " new task for execution, please retry the operation", rejected)
   }
 
-  def runningQueryError(e: Throwable): Throwable = e match {
-    case st: SparkThrowable =>
-      val errorClassPrefix = Option(st.getErrorClass).map(e => s"[$e] ").getOrElse("")
```

Comment:
I noticed that thriftserver already pretty the error class. Can we keep it when the format is PRETTY?

Reply:
The PRETTY message includes the error class, see the test
https://github.com/apache/spark/pull/37520/files#diff-44a4c34050c5ff3cc3488e7e800ad41d791936c97c2f9535371931e43868741bR165

Reply:
> ah, do you mean before this PR thriftserver print error class twice?

Yep, it did.